scripts/vellum-to-hq

```diff
@@ -226,5 +232,12 @@ def get_version(vellum_tar):
         return ""
 
 
+def byte_stream(stream):
+    # In Python 3 get a stream that accepts bytes (`sys.stdout.buffer`).
+    # In Python 2 output streams do not have a "buffer" attribute.
+    # https://github.com/amoffat/sh/issues/242
+    return getattr(stream, "buffer", stream)
```

Comment:
I generally prefer to use ```if six.PY2``` for logic that branches on python version in order to make it clear what code can be deleted post-upgrade.

Reply:
d70df42085db17de7420a77b773b2f01f49d0093